composer.json

```
@@ -46,7 +46,12 @@
         "sylius/grid": "self.version"
     },
     "conflict": {
-        "twig/twig": "^1.0"
+        "twig/twig": "^1.0",
```

```suggestion
        "twig/twig": "<2.12",
```

2.12 is the lowest tested version, I'm assuming that means it's also the lowest supported version and 2.0 thru 2.11 probably shouldn't be allowed either.